.github/workflows/merge.yml

```diff
@@ -26,33 +26,31 @@ jobs:
       - run: |
           gcloud auth configure-docker
 
-      - name: Setup defaults action java and maven properties
-        uses: actions/setup-java@v1
+      - uses: actions/setup-java@v3
+        id: maven
+        name: set java and maven cache
         with:
-          java-version: "17"
+          distribution: 'adopt'
+          java-version: '17'
           architecture: x64
+          cache: 'maven'
           server-id: forgerock-private-releases # Value of the distributionManagement/repository/id field of the pom.xml
           server-username: MAVEN_USERNAME # env variable for username in deploy
           server-password: MAVEN_CENTRAL_TOKEN # env variable for token in deploy
 
-      - name: Cache Maven packages
-        uses: actions/cache@v1
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven2-${{ hashFiles('**/pom.xml') }}
-
       - name: Build maven
         run: |
-          mvn clean install
+          mvn clean deploy -DaltDeploymentRepository=release::default::"https://${{ secrets.FR_ARTIFACTORY_USER }}:${{ secrets.FR_ARTIFACTORY_TOKEN }}@maven.forgerock.org/artifactory/community/"
```

Comment:
Why are we deploying this to artifactory? Is this necessary?

Reply:
This is the process to deploy the snapshot version of ig-extensions, until now we never publish it to forgerock maven repository, to follow the same cycle of any other projects I decided to publish that snapshot library too.